panpipes/python_scripts/run_collate_mtd_files.py

```diff
@@ -41,15 +41,42 @@
                   multimodal=args.multimodal_integration_col)
 batch_dict = {k:v.split(',') for k,v in batch_dict.items() if v is not None}
 
-# format the column names corrrectly, and add any merge columns to the dataset
+# format the column names correctly, and add any merge columns to the dataset
 for k in batch_dict.keys():
     v=batch_dict[k]
-    v =[k + ":" +v1 if k!="multimodal" else v1 for v1 in v  ]
-    batch_dict[k] = v
+    if k =="multimodal":
```

Comment:
@crichgriffin I added this piece so that the script can fetch a metadata column to calculate lisi because otherwise it wouldn't have fetched a column that it's not in the outer mudata obs ("multimodal") so now it's searching for it in every modality.

it's not the best solution as if it finds it in every modality it will eventually run lisi and plot on each one of them. (for example,for the "dataset" column, it will create and add to the batch dictionary "rna:dataset","atac:dataset" etc.  not a problem if each "dataset" in each modality encode different categories, but redundant if they don't.

could not find an easy quicker fix so leaving this here and open to suggestions. thanks!